src/style.css

```
@@ -1 +1,81 @@
 /* add styles here */
+:root {
+  --big-icon: url(/src/images/Search.svg) 26px 50% no-repeat;
+  --small-icon: url(/src/images/Search.svg) 13px 50% no-repeat;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.page {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.search_bar {
+  margin-top: 20px;
+}
+
+.search_bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, sans-serif;
+}
+
+.search_bar__input--big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  color: #3d4e61;
+  background: var(--big-icon);
+}
```

```suggestion
.search_bar__input {
  box-sizing: border-box;
  width: 100%;
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  box-shadow: 0 1px 8px 0 #3d4e611a;
  font-family: Avenir, sans-serif;
  height: 70px;
  padding-left: 62px;
  font-size: 16px;
  color: #3d4e61;
  background: var(--big-icon);
}
```

it is redundant to make modifier for big form.
make all styles in block class and just reassign in small modifier